functions/routes/ecom/modules/apply-discount.js

```diff
@@ -1,3 +1,23 @@
-exports.post = ({ appSdk }, req, res) => {
-  res.send({})
+exports.post = ({ appSdk, admin }, req, res) => {
+    /**
+   * Requests coming from the modules receive a hydrated body with two objects, `params` and application`.
```

Comment:
Acho que a frase _Requests coming from the modules receive_ ficou meio confusa :confused: 
Quer dizer, a requisição que vem do módulo (Modules API) não _recebe_ `params` and `application`, na verdade a requisição envia os dois objetos para o endpoint respectivo do app.
_Hydration_ também é uma keyword mais comum em outro contexto...

```suggestion
 /**
   * Requests coming from Modules API have two object properties on body: `params` and `application`.
```